source/_components/pi_hole.markdown

```diff
@@ -23,7 +23,7 @@ pi_hole:
 
 {% configuration %}
 host:
-  description: IP address of the host where Pi-hole is running.
+  description: IP address and Port of the host where Pi-hole is running. Pi-hole uses port 4865 by default.
```

Comment:
I'd recommend specifying here that `4865` is only the default for hass.io addons. port 80 is inferred, when the pihole is running on a raspberry pi or via the official pi-hole docker images